packages/ERTP/src/amountMath.js

```diff
@@ -167,12 +173,20 @@ const checkLRAndGetHelpers = (leftAmount, rightAmount, brand = undefined) => {
   return getHelpers(leftAmount, rightAmount);
 };
 
+/**
+ * @param {MathHelpers<Value>} h
+ * @param {Amount} leftAmount
+ * @param {Amount} rightAmount
+ * @returns {[Value, Value]}
+ */
 const coerceLR = (h, leftAmount, rightAmount) => {
   return [h.doCoerce(leftAmount.value), h.doCoerce(rightAmount.value)];
 };
 
 /** @type {AmountMath} */
 const AmountMath = {
+  // TODO: remove when the deprecated order is no longer allowed.
```

Comment:
Please include an issue number in each TODO to make it easier to find when the cleanup is done.

Reply:
Good idea